hazelcast/src/main/java/com/hazelcast/map/impl/operation/PutAllOperation.java

```diff
@@ -224,4 +235,25 @@ protected void readInternal(ObjectDataInput in) throws IOException {
     public int getClassId() {
         return MapDataSerializerHook.PUT_ALL;
     }
+
+    static class EventDescriptor {
+
+        private final Address caller;
+        private final String mapName;
```

Comment:
Minor: `caller` and `mapName` are not really necessary as those can be obtained from the operation.